src/act.wizard.cpp

```diff
@@ -4868,3 +4869,68 @@ ACMD(do_zone) {
 ACMD(do_room) {
   send_to_char(ch, "Current room num: %ld\r\n", ch->in_room->number);
 }
+
+ACMD(do_perfmon) {
+    char arg1[MAX_INPUT_LENGTH];
+
+    if (!ch->desc) {
+      log("No descriptor in do_perfmon");
+      return;
+    }
+
+    argument = one_argument(argument, arg1);
+
+    if (arg1[0] == '\0')
+    {
+        send_to_char( 
+            "perfmon all             - Print all perfmon info.\r\n"
+            "perfmon summ            - Print summary,\r\n"
+            "perfmon prof            - Print profiling info.\r\n"
+            "perfmon sect <section>  - Print profiling info for section.\r\n",
+            ch );
+        return;
+    }
+
+    if (!str_cmp( arg1, "all"))
+    {
+        char buf[MAX_STRING_LENGTH * 12];
```

Comment:
Same question as earlier about size of this buffer (and the similar declarations in this file).